app/components/Dashboard/MarketsRow.jsx

```diff
@@ -101,12 +101,8 @@ class MarketsRow extends React.Component {
 
         function getImageName(asset) {
             let symbol = asset.get("symbol");
-            if (
-                symbol === "OPEN.BTC" ||
-                symbol === "GDEX.BTC" ||
-                symbol === "RUDEX.BTC"
-            )
-                return symbol;
+            if (symbol === "OPEN.BTC" || symbol === "GDEX.BTC") return symbol;
+            if (symbol.startsWith("RUDEX.")) return symbol;
```

Comment:
This would remove the prepending RUDEX before all the RUDEX.* assets, I would not think that is appropriate.

Reply:
This is because we create rudex.xxx images for rudex tokens. There are "zoo" in styles right now.

Reply:
> This would remove the prepending RUDEX before all the RUDEX.* assets, I would not think that is appropriate.
> 
> Your comment on the PR does not mention this change.

To clarify: This change induces that rudex assets are displayed with the corresponding png file (rudex.btc -> rudex.btc.png), it still will be displayed as rudex.btc asset.

I think a generic fallback solution would be nice:

If a gateway asset is recognized (as defined in branding), it first checks if `gateway.symbol.png` exists as a symbol, and if not defaults to `symbol.png` (this is just a word description and not the best implementation.

Estimation: 1

@ruslansalikhov interested?